core/src/com/gaalf/game/precreatedEntities/balls/BasketBall.java

```diff
@@ -0,0 +1,34 @@
+package com.gaalf.game.precreatedEntities.balls;
+
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.maps.tiled.TiledMap;
+import com.badlogic.gdx.physics.box2d.CircleShape;
+import com.badlogic.gdx.physics.box2d.FixtureDef;
+import com.badlogic.gdx.physics.box2d.World;
+import com.gaalf.game.ecs.component.TextureComponent;
+import com.gaalf.game.ecs.component.TransformComponent;
+import com.gaalf.manager.GameAssetManager;
+
+import static com.gaalf.game.constants.B2DConstants.PPM;
+
+class BasketBall extends Ball{
+
+    BasketBall(String playerName, int playerNumber, TiledMap tiledMap, World world, GameAssetManager assetManager) {
+        super(playerName, playerNumber, assetManager);
+        TransformComponent transformComponent = addTransformComponent(tiledMap, 1.5f);
+        TextureAtlas textureAtlas = assetManager.manager.get(assetManager.ballSpriteAtlas);
+        Sprite sprite = textureAtlas.createSprite("Basketball");
+        TextureComponent textureComponent = addTextureComponent(sprite);
+
+        CircleShape cshape = new CircleShape();
+        cshape.setRadius(((textureComponent.sprite.getRegionWidth() * transformComponent.scale.x) / 2) / PPM);
+        FixtureDef fixtureDef = new FixtureDef();
+        fixtureDef.shape = cshape;
+        fixtureDef.density = 8;
+        fixtureDef.friction = 100.6f;
+        fixtureDef.restitution = .5f;
```

Comment:
Maybe add those values to a file as well, as was done on the other values of the ball?

Reply:
Could you elaborate?

Reply:
To me it looked like all the values of the balls where gathered in on atlas file. Maybe we should gather all these values in one place as well, so we dont need to hard code them in every time

Reply:
The atlas file only covers details about the placement of a texture in an atlas file, not box2d values

Reply:
ah, ok. I don't have too much experience with atlas files. But I still think it is a good idea to gather all the samey values in one place. But maybe in an other pull request?

Reply:
Yeah, that's a refactor waiting to happen.